examples/cjs/cjs.py

```diff
@@ -57,24 +65,184 @@ def model(data):
                                 infer={"enumerate": "parallel"})
                 mu_y_t = rho * z
                 pyro.sample("y_{}".format(t), dist.Bernoulli(mu_y_t),
-                            obs=data[:, t])
-            first_capture_mask |= data[:, t].byte()
+                            obs=capture_history[:, t])
+            first_capture_mask |= capture_history[:, t].byte()
+
+
+"""
+In our second model variant there is a survival probability phi_t for 6
```

Comment:
nit: consider saying "there is a time-varying survival probability"

Reply:
done